packages/zkdb/src/sdk/impl/database.ts

```diff
@@ -87,16 +88,6 @@ export class ZKDatabaseImpl implements ZKDatabase {
     return result.unwrap().map((collection) => collection.name);
   }
 
-  async create(merkleHeight: number, publicKey: PublicKey): Promise<boolean> {
```

Comment:
Can you explain to me why you removed this function?

Reply:
We have two exact function, one in globalContext